Classes/Domain/Model/Metadata.php

```diff
@@ -30,7 +34,9 @@ class Metadata extends \TYPO3\CMS\Extbase\DomainObject\AbstractEntity
     protected $indexName;
 
     /**
-     * @var int
+     * @var \TYPO3\CMS\Extbase\Persistence\ObjectStorage<\Kitodo\Dlf\Domain\Model\MetadataFormat>
```

Comment:
This IRRE-relation was not correct. I'm not fully sure about the lazy-loading. Please @dvoracek-slub, have a look on it, if we need the LazyLoadingProxy here.

Reply:
IMO, this should be fine. The LazyLoadingProxy is only used for single objects (`_:1` relations); for collections, their `LazyObjectStorage` inherits from `ObjectStorage`, so a conversion isn't necessary.

Reply:
Ok. Thank you for clarifying.